src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendanceController.kt

```diff
@@ -30,9 +31,14 @@ class AttendanceController(private val attendanceService: AttendanceService) {
       @ApiParam(value = "Attendance details", required = true)
       @RequestBody
       @Valid
-      attendance: CreateAttendanceDto): AttendanceDto {
+      attendance: CreateAttendanceDto): ResponseEntity<Any> {
+    val createdAttendance: AttendanceDto =  try {
+      attendanceService.createAttendance(attendance)
+    } catch (e: AttendanceExists) {
+      return ResponseEntity.badRequest().build()
```

Comment:
Should we return a http 409?

Reply:
We definitely should, good catch!

Reply:
Might be worth returning an error message too.

Reply:
👍